docgen/src/guides/angular-integration.md

```diff
@@ -0,0 +1,366 @@
+
+At the end of this guide you will have an Angular2+ application running with an InstantSearch.js search experience!
+
+For the rest of the guide we assume that you have some Angular2 knowledge and the angular-cli installed on your machine. If it's not the case have a look at the awesome lessons on [egghead.io](https://egghead.io/lessons/angular-2-say-hello-world-to-angular-2).
+
+If you are new to InstantSearch.js, best is to read first the [getting started](https://community.algolia.com/instantsearch.js/v2/getting-started.html) guide.
+
+Before going through this guide, it's well recommended to read the [customize widgets](https://community.algolia.com/instantsearch.js/v2/guides/customization.html) and the [introduce to connectors](https://community.algolia.com/instantsearch.js/v2/connectors.html) guides for InstantSearch.js
+
+## Create an Angular app
+
+We will create a default Angular application using the angular-cli, this is done like this:
+
+```sh
+$ ng new [project-name]
+$ cd [project-name]
+```
+
+Later, we can serve the application with `$ ng serve`.
+
+## Install InstantSearch.js
+
+First of all, you need to add InstantSearch.js to your dependencies:
+
+```sh
+$ yarn add instantsearch.js
+// OR
+$ npm install instantsearch.js --save
+```
+
+After you have installed InstantSearch.js to your application dependencies, we need to instantiate the InstantSearch.js library into an Angular service. So we will generate this new service:
+
+```sh
+$ ng generate service services/instantsearch
+// OR
+$ ng g s services/instantsearch
+```
+
+Open the new generated file `src/app/services/instantsearch.service.ts` and here initialize InstantSearch.js
+
+In order to initialize InstantSearch.js, you need an Algolia account with a configured and non-empty index. For the sake of simplicity, let's use the credentials of our test index:
+
+```javascript
+import { Injectable } from '@angular/core';
+import instantsearch from 'instantsearch.js/es';
+
+@Injectable()
+export class InstantSearchService {
+  search = instantsearch({
+    appId: 'latency',
+    apiKey: '3d9875e51fbd20c7754e65422f7ce5e1',
+    indexName: 'bestbuy',
+    urlSync: true
+  });
+
+  constructor() {}
+}
+```
+
+`appId`, `apiKey` and `indexName` are mandatory. Those values are credentials of your application in Algolia. They can be found in your [Algolia dashboard](https://www.algolia.com/api-keys).
+
+You can see the code example above we are importing the ES6 module build of the InstantSearch.js library. This will reduce a lot the final build size of your application (this is known as [tree shaking](https://webpack.js.org/guides/tree-shaking/) in the JavaScript world).
+
+Don't forget to register your service into the providers of your `src/app.modules.ts` file to be able to inject it into your next components. [(Official Angular documentation)](https://angular.io/tutorial/toh-pt4#inject-the-heroservice).
```

Comment:
Nope, that was the issue you went into the first time of the service not accessible.